lib/tpm2_session.c

```diff
@@ -86,6 +86,14 @@ void tpm2_session_set_nonce_caller(tpm2_session_data *data, TPM2B_NONCE *nonce)
     data->nonce_caller = *nonce;
 }
 
+UINT16 tpm2_session_get_nonce_tpm_size(tpm2_session *session) {
+    return session->internal.nonceNewer.size;
```

Comment:
You only need one of these functions. And that's one that returns a pointer to the nonceNewer struct.

Reply:
Fixed